src/views/ManageAssets.vue

```diff
@@ -105,6 +117,35 @@ export default {
   flex-direction: column;
   min-height: 0;
 
+  &_noneTop {
+    h5 {
+      color: $color-primary;
+      &:hover {
+        cursor: pointer;
+      }
+    }
+  }
+
+  &_noneText {
+    font-weight: bold;
+  }
+
+  &_customText {
+    font-weight: 100;
+  }
+
+  &_noneBottom {
```

Comment:
shouldn't use fixed height
we would need to adjust this value anytime we change the content - not good

This container should always be flex such that the bottom sticks